### PR TITLE
Add cluster_shutdown module

### PR DESCRIPTION
**Pull request by PolonaM**
Added cluster_shutdown module

### CODE REVIEW

File: plugins/modules/cluster_shutdown.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+description:
+  - Shutdown cluster.
+version_added: 1.2.0
+extends_documentation_fragment:
+  - scale_computing.hypercore.cluster_instance
+seealso: []
+options:
+  force_shutdown:
+      description:
+        - Bypasses attempts to gracefully shut down all runnning VirDomains before system shutdown.
```
Comment (justinc1) [inline plugins/modules/cluster_shutdown.py:27]: This means all running VMs will be force shutdown? The host/node will not be force shutdown?

File: plugins/modules/cluster_shutdown.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+
+DOCUMENTATION = r"""
+module: cluster_shutdown
+
+author:
+  - Polona Mihalič (@PolonaM)
+short_description: Shutdown cluster.
+description:
+  - Shutdown cluster.
```
Comment (justinc1) [inline plugins/modules/cluster_shutdown.py:19]: Add to description if module shut down a single node, or all nodes in a cluster.

**Review by justinc1 — CHANGES_REQUESTED** [review]
Currently only clarifications in `DOCUMENTATION` are needed.

File: plugins/modules/cluster_shutdown.py (hunk 2 above)
Comment (PolonaM) [reply plugins/modules/cluster_shutdown.py:19]: In the documentation it says: "Shutdown the system.". And we have only one node in each cluster, so I can't really test if really all nodes do shut down. We should reconfigure our system to have one cluster with two nodes to be able to test this. But we can also trust the documentation :)

Comment (justinc1) [reply plugins/modules/cluster_shutdown.py:19]: Or we ask in slack, and then amend to description if needed.

**Review by justinc1 — APPROVED** [review]

**Pull request merged**